2-patterns_integrations/src/main/java/com/epam/dhontar/aqamp/api/RestClient.java

```diff
@@ -0,0 +1,39 @@
+package com.epam.dhontar.aqamp.api;
+
+import static com.epam.dhontar.aqamp.utils.enums.ServicesEndpoints.BASE_URL;
+
+import static io.restassured.RestAssured.given;
+
+import com.epam.dhontar.aqamp.utils.enums.ServicesEndpoints;
+
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+
+public class RestClient {
+    private String url;
```

Comment:
Create here private static final String URL with your actual url and use it everywhere in this class. Don't need to transmit it outside

Reply:
I do not actually get the idea. This URL defers according to the endpoint that is passed as a constructor argument.